accounts/migrations/0008_auto_20170331_1702.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
I thought django always provided backward and forward migrations...? Shouldn't this be enough in almost all cases?